.env.sample

```
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
```

```suggestion
MYSQLDB_USER=
```

remove other properties, it should be like template